engineioxide/src/transport/polling/payload/encoder.rs

```diff
@@ -7,6 +7,8 @@
 //!    * binary encoder (used when there is binary packets and the client supports binary)
 //!
 
+#[cfg(feature = "v3")]
+use bytes::{BufMut, BytesMut};
```

Comment:
Feature specific imports are more often put in the specific functions in this module. Would it be possible to uniform this?

Reply:
Heh, yes and no -- `BytesMut` is an argument to the function that's conditionally compiled, so it needs to be resolvable before the body of the function.  But can just use a fully-qualified name there, and move the trait import inside the function.